src/components/SQForm/SQFormDatePicker.tsx

```diff
@@ -8,7 +8,7 @@ import {
   ClickAwayListener,
   InputBaseComponentProps,
 } from '@material-ui/core';
-import BaseFieldProps from '../../types/BaseFieldProps';
+import type BaseFieldProps from '../../types/BaseFieldProps';
```

Comment:
[nit/suggestion] it'd be snazzy to move these to use absolute import paths. `'../../types/'` vs `'types/'`